R/revdep.R

```diff
@@ -1,11 +1,11 @@
 #' Reverse dependency checks
 #'
-#' Performs set up for checking the reverse dependencies of an R package, as
+#' Performs setup for checking the reverse dependencies of an R package, as
 #' implemented by the revdepcheck package:
-#' * Creates `revdep/` directory and adds it to `.Rbuildignore`
+#' * Creates `revdep/` directory and adds it to `.Rbuildignore`.
 #' * Populates `revdep/.gitignore` to prevent tracking of various revdep
-#' artefacts
-#' * Prompts user to run the checks with `revdepcheck::revdep_check()`
+#' artefacts.
+#' * Prompts the user to run the checks with [revdepcheck::revdep_check()].
```

Comment:
I can't link to a topic in a package that's not on CRAN, which revdepcheck is not.

Reply:
Why can't you link to it? Is there a CRAN rule forbidding this?